workflows/clean-era5-workflow.yaml

```diff
@@ -11,9 +11,9 @@ spec:
   arguments:
     parameters:
     - name: in-zarr
-      value: "clean-dev/ERA-5/tmax.1995-2014.0p25.zarr"
+      value: "clean-dev/ERA-5/F320/dtr.1995-2015.F320.v2.zarr"
     - name: out-zarr
-      value: "scratch/clean-dev/ERA-5/tasmax.1995-2014.0p25.zarr"
+      value: "scratch/clean-dev/ERA-5/dtr.1995-2015.F320.zarr"
```

Comment:
Same comment as above, could we have this default value point to the tasmax zarr store instead of dtr while we're still prototyping?

Reply:
yes definitely - I actually forgot that I'd switched this to dtr and meant to switch it back